Modules/IO/ImageBase/test/itkUnicodeIOTest.cxx

```diff
@@ -84,8 +84,8 @@ int main( int , char * [] )
 
   // Put alpha.txt encoded in utf8 within a std::string
   std::string utf8_str;
-  utf8_str.append(1, (char)(0xCE));
-  utf8_str.append(1, (char)(0xB1));
+  utf8_str.append(1, static_cast<char>(std::uint8_t{ 0xCE }));
+  utf8_str.append(1, static_cast<char>(std::uint8_t{ 0xB1 }));
```

Comment:
This looks like is should work, but there has to be a more concise where to do this!

Reply:
@blowekamp 
> This looks like is should work, but there has to be a more concise where to do this!

Hmmm... maybe somethis like this:

    utf8_str.append(1, '\xCE');
    utf8_str.append(1, '\xB1');

What do you think?

Reply:
That is what I would like to think would work. Hard to say if a compiler would warn. 🤷‍♂

Reply:
https://en.cppreference.com/w/cpp/language/string_literal

Do we need a prefix on the string literal?

Reply:
@hjmjohnson Thanks for the link, Hans, but I don't think such a prefix is necessary. The '\xnn' escape sequence appears to do the job: https://en.cppreference.com/w/cpp/language/escape

Reply:
@blowekamp 
> That is what I would like to think would work. Hard to say if a compiler would warn. 

I just checked at https://rextester.com/VMDIR6746 with Warning Level 4 (`/W4`):

    str.append(1, (char)0xCE);  // <== warning: cast truncates constant value

    str.append(1, '\xCE');  // OK, no warning  :-)

Reply:
Great work figuring out these details! That link was very helpful Hans. I believe the original code was declaring a signed integer the casting it to char. The single quote syntax directly defines a char type!

Reply:
I just checked: `0xCE` is equal to `206`, while the integer value of `(char)0xCE` is `-50`, by default, for MSVC (having signed `char` by default). But yes, `(char)0xCE == \xCE` is true.